src/shared/lib/StoreUtils.ts

```diff
@@ -1873,8 +1874,9 @@ export function filterAndAnnotateMutations(
             mutation,
             getPutativeDriverInfo(mutation)
         ); // annotate
-        annotatedMutation.hugoGeneSymbol =
-            entrezGeneIdToGene[mutation.entrezGeneId].hugoGeneSymbol;
+        annotatedMutation.hugoGeneSymbol = (
+            entrezGeneIdToGene[mutation.entrezGeneId] || mutation.gene
+        ).hugoGeneSymbol;
```

Comment:
Same workaround as above. Similar to the one above, should we always rely on `mutation.gene` instead of referring to `entrezGeneIdToGene`?